nix/package.nix

```diff
@@ -13,7 +13,7 @@
 }:
 
 let
-  qgis-unwrapped = libsForQt5.callPackage ./unwrapped.nix {
+  qgis-unwrapped = qt6Packages.callPackage ./unwrapped.nix {
     withGrass = withGrass;
     withServer = withServer;
     withWebKit = withWebKit;
```

Comment:
Seems like this is a stale argument after fe354cc977d7c349a0bc17070f02246fe15b25d1

Reply:
Great thanks. You are right with this change.